staging/src/k8s.io/client-go/testing/fixture.go

```diff
@@ -100,6 +100,14 @@ func ObjectReaction(tracker ObjectTracker) ReactionFunc {
 				return true, nil, err
 			}
 			if action.GetSubresource() == "" {
+				if action.Tracked {
+					// Return an tracked object instead of a newly created one (required for TokenReviews)
+					// for backward compatibility, if no such object, return the created one.
+					obj, err := tracker.Get(gvr, ns, objMeta.GetName())
```

Comment:
this doesn't seem correct... for something like TokenReview, I would expect tests to add a reactor and intercept the object and manipulate it however they want, not rely on a persisted tracked object

does adding a reactor work?

/hold

Reply:
reactor works and solves the problem. 
No need for the suggested change.